src/bank.rs

```diff
@@ -1087,6 +1125,11 @@ impl Bank {
             .unwrap_or(0)
     }
 
+    /// TODO: fix once there's real staking
```

Comment:
Can you please describe what needs to be "fixed" here.  Or better, reference a github issue with those details